static/src/stylesheets/module/_adslot.scss

```diff
@@ -69,8 +69,6 @@
     .sticky-ad-banner & {
         z-index: 1022;
         width: 100%;
-        transform: translateZ(0);
-        will-change: transform;
```

Comment:
In this case the element is already promoted to its own layer because of `position: fixed`. `will-change: transform` was a lie because we never changed transform.